### DISCUSSION TITLE
WinForms web page padding

### DISCUSSION THREAD

**Discussion by raphm4044** [cefsharp/CefSharp#4105] [General]
- **What version of the product are you using?**
    CefSharp v101.0.180, CefSharp.WinForms v101.0.180

- **What architecture x86 or x64?**
    x64
	
- **What version of .Net?**
    .NET Framework 4.8
    
- **On what operating system?**
    Windows 11 build 22000.675

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    WinForms
    
- **What steps will reproduce the problem?** -
    Create a new WinForms app, Install CefSharp.WinForms and add a ChromiumWebBrowser and load a website

- **What is the expected output? What do you see instead?**
    A padding
- **Please provide any additional information below.**
    
![image](https://user-images.githubusercontent.com/81994075/168465206-71ae31ea-2d4d-4871-a2ca-cb6581927c83.png)

- **Does this problem also occur in the `CEF` Sample Application**
    No

**Comment by amaitland** [comment]
Converting to discussion as this is unlikely a bug.

Please confirm you've tested with the [MinimalExample](https://github.com/cefsharp/CefSharp.MinimalExample).

What `DPI` are you using? Make sure your application is `DPI` aware https://github.com/cefsharp/CefSharp/wiki/General-Usage#winforms-high-dpi

**Reply by raphm4044** [reply]
Im using 125% DPI because i have a 1920x1080 laptop screen.

**Reply by amaitland** [reply]
Then you need to make your application `DPI aware`. This is the most common cause of display issues.

**Reply by raphm4044** [reply]
what

**Reply by amaitland** [reply]
https://docs.microsoft.com/en-us/dotnet/desktop/winforms/high-dpi-support-in-windows-forms?view=netframeworkdesktop-4.8#configuring-your-windows-forms-app-for-high-dpi-support
